grafana-plugin/src/state/plugin/plugin.ts

```diff
@@ -274,30 +276,31 @@ export class PluginState {
     }
 
     // Step 3. reprovision the Grafana plugin settings, storing information that we get back from OnCall's backend
-    try {
-      const {
-        grafanaToken,
-        onCallAPIResponse: { onCallToken: onCallApiToken, ...jsonData },
-      } = pluginInstallationOnCallResponse;
-
-      await this.updateGrafanaPluginSettings({
-        jsonData: {
-          ...jsonData,
-          onCallApiUrl,
-        },
-        secureJsonData: {
-          grafanaToken,
-          onCallApiToken,
-        },
-      });
-    } catch (e) {
-      return this.getHumanReadableErrorFromGrafanaProvisioningError(
-        e,
-        onCallApiUrl,
-        errorMsgVerb,
-        onCallApiUrlIsConfiguredThroughEnvVar
-      );
-    }
+    // try {
```

Comment:
Hack to workaround existing code to not overwrite what is configured by backend plugin.